### PR TITLE
Add weight into qiskit max cut calculation

### PR DESCRIPTION
**Pull request by Elscrux**

### CODE REVIEW

File: qiskit/max-cut/maxCut_qiskit.py [hunk 1]
```diff
@@ -31,6 +31,25 @@
 input_path = sys.argv[1]
 output_path = sys.argv[2]
 
+# To include the weight information, we'll need to map the weight to the label
+# so the parser can read it. Therefore, remove all existing "label" lines and
```
Comment (schweikart) [inline qiskit/max-cut/maxCut_qiskit.py:35]: does this code really remove label lines? where?

File: qiskit/max-cut/maxCut_qiskit.py [hunk 2]
```diff
@@ -31,6 +31,25 @@
...
+
+    # Iterate through the lines, removing lines with "label" and replacing "weight" with "label"
+    for line in lines:
+        if "label" not in line:
+            modified_line = line.replace("weight", "label")
+            file.write(modified_line)
+
+    # Truncate the remaining content in case the new content is shorter than the old content
+    file.truncate()
+
```
Comment (schweikart) [inline qiskit/max-cut/maxCut_qiskit.py:52]: This seems a bit unnecessary. Why don't you just use the `weight` attribute?

**Review by schweikart — CHANGES_REQUESTED** [review]

File: qiskit/max-cut/maxCut_qiskit.py (hunk 1 above)
Comment (Elscrux) [reply qiskit/max-cut/maxCut_qiskit.py:35]: Yes, it does so by skipping the lines with "label" in it in the if down below.

File: qiskit/max-cut/maxCut_qiskit.py (hunk 2 above)
Comment (Elscrux) [reply qiskit/max-cut/maxCut_qiskit.py:52]: The parser library that we use apparently can just parse the label but no custom attributes like our weight.

Comment (schweikart) [reply qiskit/max-cut/maxCut_qiskit.py:52]: That's weird :/

**Comment by schweikart** [comment]
Rebased onto develop

**Review by schweikart — APPROVED** [review]

**Pull request merged**